triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -67,8 +71,62 @@ object TriggerServiceFixture {
     val host = InetAddress.getLoopbackAddress
     val isWindows: Boolean = sys.props("os.name").toLowerCase.contains("windows")
 
-    // Launch a toxiproxy instance. Wait on it to be ready to accept
-    // connections.
+    // Set up an authentication service if enabled
+    val authServiceAdminLedger: Future[Option[(SandboxServer, Port)]] =
+      if (!auth) Future(None)
+      else {
+        val adminLedgerId = LedgerId("admin-ledger")
+        for {
+          ledger <- Future(
```

Comment:
You’re not actually enabling auth are you?

Reply:
No auth on the admin ledger. It is enabled on the main ledger if the `auth` flag is set

Reply:
Does the auth service support running against a ledger with auth enabled? If not, I guess that obsoletes my comment about whether we can share sandbox for both.

Reply:
Yeah I was just going to mention that in your comment above. Trying to find that out now

Reply:
Apparently the auth service will be authenticated against the ledger it's providing auth for - I'm finding out how that works later today

Reply:
Ok no, the auth service cannot grant itself access to the main ledger. It needs the separate admin ledger unauthenticated